addons/binding/org.openhab.binding.nest/src/test/java/org/openhab/binding/nest/internal/data/GsonParsing.java

```diff
@@ -0,0 +1,308 @@
+package org.openhab.binding.nest.internal.data;
+
+import static org.junit.Assert.*;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+import org.junit.Test;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class GsonParsing {
```

Comment:
This test is / was also somewhere else in a separate binding, can this one be removed?